amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/dashboard/PaimonTableDescriptor.java

```diff
@@ -525,14 +525,19 @@ public List<TagOrBranchInfo> getTableBranches(AmoroTable<?> amoroTable) {
   @Override
   public List<ConsumerInfo> getTableConsumerInfos(AmoroTable<?> amoroTable) {
     FileStoreTable table = getTable(amoroTable);
+    FileStore<?> store = table.store();
     ConsumerManager consumerManager = new ConsumerManager(table.fileIO(), table.location());
     List<ConsumerInfo> consumerInfos = new ArrayList<>();
     try {
       consumerManager
           .consumers()
           .forEach(
-              (consumerId, nextSnapshot) ->
-                  consumerInfos.add(new ConsumerInfo(consumerId, nextSnapshot)));
+              (consumerId, nextSnapshotId) -> {
+                Snapshot snapshot = table.snapshotManager().snapshot(nextSnapshotId - 1);
```

Comment:
Will there be any exception when nextSnapshotId=1?

Reply:
If first snapshot is < 0, may be have exp. Flink Checkpoint start id is 0